tracer/src/Datadog.Trace/Tags.cs

```diff
@@ -706,6 +706,12 @@ internal static class Propagated
             /// lower-case hexadecimal string with no zero-padding or `0x` prefix.
             /// </summary>
             internal const string TraceIdUpper = "_dd.p.tid";
+
+            /// <summary>
+            /// Tag used to propagate the unsigned  64 bits last parent Id
+            /// lower-case 16 characters hexadecimal string
+            /// </summary>
+            internal const string LastParentId = "_dd.parent_id";
```

Comment:
Wasn't this defined already in a previous PR? (and is it used anywhere?)

Reply:
So no, I tried looking around and only found [this ](https://github.com/search?q=repo%3ADataDog%2Fdd-trace-dotnet%20%22_dd.parent_id%22&type=code) within the `SpanMessagePackFormatter`, but for me to be able to exclude it from the Snapshots like all others examples I needed to add it here under `Tags.cs` .

`private readonly byte[] _lastParentIdBytes = StringEncoding.UTF8.GetBytes("_dd.parent_id");`

Reply:
Ok, thanks! Do you mind updating that line in `SpanMessagePackFormatter` to use the constant as well?